python/ray/data/_internal/compute.py

```diff
@@ -341,12 +345,16 @@ def map_block_nosplit(
                 ):
                     block, meta = blocks_in.pop()
                     if context.block_splitting_enabled:
-                        ref = worker.map_block_split.remote(
-                            block,
-                            meta.input_files,
-                            *fn_args,
-                            **fn_kwargs,
+                        raise NotImplementedError(
+                            "DatasetContext.block_splitting_enabled currently "
+                            'not supported with compute="actors"'
                         )
+                        # ref = worker.map_block_split.remote(
+                        #     block,
+                        #     meta.input_files,
+                        #     *fn_args,
+                        #     **fn_kwargs,
+                        # )
                     else:
```

Comment:
Can we just comment out this `if` branch for now, instead of throwing an exception? This would make CI test a lot of easier, and we can add to documentation that dynamic block split does not support datasets actor mode later.

Reply:
Line 375-379 of this method is also needed to be commented out.

Reply:
Makes sense, thanks!